api/config/config_local.json

```diff
@@ -50,5 +50,11 @@
   "moodle": {
     "host": "aoudev.nnlm.gov",
     "enableMoodleBackend": true
+  },
+  // The access object specifies whether each of the following access requirements block access
```

Comment:
Thoughts on comment placement: I wonder if it might be easier to maintain this comment as a single code-comment in WorkbenchConfig.java, rather than replicated across each of the config files here.